### ISSUE TITLE
train_list.txt 

### ISSUE THREAD

**Issue by gtgtgt1117** [skertiver/scene_classification#1]
hello @skertiver 

could you tell me where is the train_list and val_list?thank you

**Comment by skertiver** [comment]
hi,the training set and the validation set need to be prepareed before training，you can download the dataset from http://places2.csail.mit.edu/download.html.

**Comment by gtgtgt1117** [comment]
> hi,the training set and the validation set need to be prepareed before training，you can download the dataset from http://places2.csail.mit.edu/download.html.



thank you 
could you provide the inference code?
